EventFilter/L1ScoutingRawToDigi/python/ScCaloRawToDigi_cfi.py

```diff
@@ -1,7 +1,26 @@
 import FWCore.ParameterSet.Config as cms
 
+# default parameters for the DMA input source
+dmaDataModeParameters = cms.PSet(
+  dataSourceMode=cms.string("DMA"),
+  sourceId=cms.int32(2)
+)
+
+# default parameters for the TCP source
+tcpDataModeParameters = cms.PSet(
+  dataSourceMode=cms.string("TCP"),
+  jetSourceIdList = cms.vint32(22),
+  eGammaSourceIdList = cms.vint32(23),
+  tauSourceIdList = cms.vint32(25),
+  etSumSourceIdList = cms.vint32(24),
+)
+
 ScCaloUnpacker = cms.EDProducer("ScCaloRawToDigi",
   srcInputTag = cms.InputTag("rawDataCollector"),
+  # DMA / TCP mode configs
+  dataSource = tcpDataModeParameters,
+  # unpack the full set of energy sums
+  enableAllSums = cms.bool(True),
   # print all objects
   debug = cms.untracked.bool(False)
-)
+)
```

Comment:
can you add newline to the end of this file?
Since recently there was a PR adding them back to files that didn't have them.

Reply:
Hi Srecko, done. 
Most likely a dummy question, but why isn't this action performed when running `scram build code-format`?

Reply:
good point, but no idea about code-format.